client/app/dbaas/logs/detail/options/logs-options.constants.js

```diff
@@ -0,0 +1,8 @@
+angular.module("managerApp").constant("LogOptionConstant", {
+    orderBaseURL: {
+        US: "https://ovh.us/order/express/#/express/review",
```

Comment:
OrderHelper should already do that for you :)

from OrderHelper buildUrl function => 

`return this.User.Lexi().get()
            .$promise
            .then(user => {
                let targetURL;
                switch (user.ovhSubsidiary) {
                    case "FR":
                        targetURL = "https://www.ovh.com/fr";
                        break;
                    case "ASIA":
                        targetURL = "https://ca.ovh.com/asia";
                        break;
                    case "CA":
                        targetURL = "https://ca.ovh.com/en";
                        break;
                    case "SG":
                        targetURL = "https://ca.ovh.com/sg";
                        break;
                    case "WS":
                        targetURL = "https://us.ovh.com/es";
                        break;
                    case "WE":
                        targetURL = "https://ca.ovh.com/en";
                        break;
                    case "QC":
                        targetURL = "https://ca.ovh.com/fr";
                        break;
                    default:
                        targetURL = "https://www.ovh.com/fr";
                }

                return `${targetURL}${path}`;
            });`

Reply:
And github won't format my code correctly.  I don't know why and I'm sorry

Reply:
Thanks a lot for enlightening me on this service :).
I guess this service is not being used anywhere and I did not know about it. I was able to use this service with small changes to it.